client/src/components/map-page/MapDisplay.js

```diff
@@ -29,7 +28,7 @@ export default function MapDisplay({ handleStarClick, parkData }) {
       }}
       mapStyle="mapbox://styles/trix2705/cko8hkfyn0sm617o9ek7qmkiq"
     >
-      {parkData.length > 0 &&
+      {parkData !== 0 &&
```

Comment:
Wouldn't this suffice?
```suggestion
      {parkData &&
```